### DISCUSSION TITLE
flash-cat

### DISCUSSION THREAD

**Discussion by h7899** [firemodels/fds#13052] [Q&A]
If I can, I'd like to seek an fds example file that uses the FLASH-CAT model to simulate a cable fire

**Comment by mcgratta** [comment]
Utilities/Training/NRC-EPRI/

has some training cases.

**Comment by ayub766** [comment]
> If I can, I'd like to seek an fds example file that uses the FLASH-CAT model to simulate a cable fire

**Comment by mcgratta** [comment]
[This input file](https://github.com/mcgratta/fds/blob/master/Utilities/Training/NRC_EPRI/FDS_Input_Files/Switchgear_Room_Cabinet_v6.fds) is documented in a [US Nuclear Regulatory Commission report](https://www.nrc.gov/reading-rm/doc-collections/nuregs/staff/sr1934/index.html).
